featureflags/evaluations/auth_target.py

```diff
@@ -66,16 +67,25 @@ def from_dict(cls: Type[T], src_dict: Dict[str, Any]) -> T:
         return target
 
     def get_attr_value(self, attribute: str) -> Optional[str]:
+        if not attribute:
+            log.debug("Attribute is empty")
+            return None
         result: Any = getattr(self, attribute, None)
-        if result is None and not isinstance(self.attributes, Unset):
+        if not result and not isinstance(self.attributes,
```

Comment:
I need to remove this Unset class, because api generator creates this class for values not presented in response. So I will need to create separate ticket for removing unset and UNSET values